python/lsst/ip/diffim/dipoleFitTask.py

```diff
@@ -1105,25 +1096,33 @@ def measure(self, measRecord, exposure, posExp=None, negExp=None):
         result : TODO: DM-17458
             TODO: DM-17458
         """
-
         result = None
         pks = measRecord.getFootprint().getPeaks()
 
         # Check if the footprint consists of a putative dipole - else don't fit it.
         if (
                 # One peak in the footprint (not a dipole)
-                (len(pks) <= 1)
+                ((nPeaks := len(pks)) <= 1)
                 # Peaks are the same sign (not a dipole)
-                or (len(pks) > 1 and (np.sign(pks[0].getPeakValue())
+                or (nPeaks > 1 and (np.sign(pks[0].getPeakValue())
```

Comment:
Are the peaks guaranteed to be ordered from largest positive peak to largest negative peak? I am concerned about checking pks[0] and pks[-1] explicitly, instead of sorting the peaks. If the peaks *are* sorted, please add a code comment stating that assumption.

Reply:
Yes, our convention is that peaks are always ordered from highest to lowest. This is a pretty deep convention and a *lot* of code depends on it.